### ISSUE TITLE
No list of existing locations

### ISSUE THREAD

**Issue by SkyNomad** [DeepskyLog/DeepskyLog#472]
When entering a new location in DeepskyLog, you only see your own existing locations, not those created by others. This causes a lot of double locations. It would be nice if the map would show all existing locations, so you could pick one.

**Comment by WimDeMeester** [comment]
I don't think you should see the locations of all other observers. This will make everything very slow and has no added value (according to me). Every observers has a private list of locations anyway. What we should change, is a way not to show all locations in the drop down menu when we are searching for an observation.

**Comment by TomCorstjens** [comment]
I do not agree, especially when making common sessions with other observers: this works when you share the same observing location. In case one has a different naming of that location (e.g. Grandpré, Grand Pre, GrandPré, Grandpre, ...) this is confusing.

**Comment by WimDeMeester** [comment]
I think it is even more confusing to have the same name for one location. For example, I want to call my observation place at home 'My garden'. I invite some people to come along for an observing night. I don't think they want to see 'My garden' as the location of the session. We still have to think about this, maybe we can discuss it at the meeting.

**Comment by WimDeMeester** [comment]
Trying to think about a good solution for the laravel version of DeepskyLog. @TomCorstjens , please also think how we can solve this.

**Comment by WimDeMeester** [comment]
I don't know if we are allowed to show the locations of the other users (GDPR and stuff)... Can someone who knows more about GDPR comment on this?
